README.md

```diff
@@ -84,32 +84,35 @@ primary language is not English, you may consider also importing the English ver
 
 ## Basic Usage ##
 
-1.  Click on the ![](https://foosoft.net/projects/yomichan/img/yomichan-icon.png) icon in the browser toolbar to open the Yomichan actions dialog.
+1.  Click the <img src="ext/mixed/img/yomichan-icon.svg" alt="" width="16" height="16"> _Yomichan_ button in the browser bar to open the quick-actions popup.
```

Comment:
Why raw HTML?

Reply:
Forcing the width/height to 16, since they are SVGs. Should prevent any layout issues that could occur if these sizes are ever changed. It should also make their load placeholder have a constant size and not change the layout after the content is loaded.